src/cljs/main/broadfcui/page/workspace/data/tab.cljs

```diff
@@ -131,13 +132,9 @@
                  :value (utils/get-access-token)}]
         [:input {:type "hidden"
                  :name "attributeNames"
-                 :value (->> (persistence/try-restore
-                              {:key (str (common/workspace-id->string
-                                          workspace-id) ":data:" selected-entity-type)
-                               :initial (constantly {})})
-                             :column-meta
+                 :value (->> (or (:visible-columns @state) ((@refs "entity-table") :get-ordered-columns))
```

Comment:
I think we shouldn't keep a copy of the column state in this component, and just always ask for it via `:get-ordered-columns` when needed here.